packages/docs/package.json

```diff
@@ -25,17 +25,15 @@
     "gatsby-plugin-sass": "^5.11.1",
     "gatsby-plugin-sitemap": "^5.11.1",
     "gatsby-source-filesystem": "^4.11.1",
-    "react-helmet": "^6.1.0",
-    "sass": "^1.50.0"
-  },
-  "devDependencies": {
-    "@types/node": "^17.0.23",
-    "typescript": "^4.6.3"
+    "react-helmet": "^6.1.0"
   },
   "peerDependencies": {
+    "@types/node": "^17.0.22",
     "@types/react": "^17.0.10",
     "@types/react-dom": "^17.0.10",
     "react": ">=16.0.0",
-    "react-dom": ">=16.0.0"
+    "react-dom": ">=16.0.0",
+    "sass": "^1.43.4",
+    "typescript": "^4.5.4"
```

Comment:
I don't think we can technically put this one here in `peerDependencies` because the root-level package file doesn't have it as a dependency. If it is moved back, does it break things?

Reply:
I don't think it will break anything, but I'll confirm